src/Badge.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space } from 'styled-system'
+import { theme } from '../'
+
+const type = props => {
+  switch(props.type) {
```

Comment:
We should probably use the styled-system `color` function here, but would love to hear some thoughts. Eg. `<Badge bg='blue' />` and `<Badge bg='green' />` seems more readable in code to me and it would be consistent with other components like `Box`. If we wanted non-presentational aliases, those *could* be added in the theme, but don't think they're really necessary

Reply:
So @Degron actually brought that up to me. I wasn't sure if we want to approach it that way or not. I suggested taking this approach because there seem to be only certain types of badges so we ensure they have the right color and background based on their type. Unless it's not a good idea to enforce it or it's not meant to do it.

Reply:
My thinking is that for communication between designers and developers, it's generally easier to refer to is as "the blue badge" instead of "the information badge" – e.g. presentational semantics over functional. Generally I don't think people will use the wrong color, and it should be up to the designers to ensure that color is used as intended across the UI